common/buildcraft/lib/misc/StackUtil.java

```diff
@@ -17,10 +15,11 @@
 
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
+import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTBase;
 import net.minecraft.nbt.NBTTagCompound;
 import net.minecraft.util.NonNullList;
-
```

Comment:
Please don't remove import spacing.

(There's a list of import order here: https://github.com/BuildCraft/BuildCraft/blob/8.0.x-1.12.2/guidelines/buildcraft.checkstyle#L15, where different sections should be separated).

Reply:
Sorry, I didn't notice that. IDEA automatically changed it.